core/templates/pages/certificate-assessment-player-page/certificate-assessment-player-page.component.html

```
@@ -1,30 +1,20 @@
-                                                  [currentQuestion]="getCurrentQuestion()"
-                                                  [currentQuestionIndex]="currentQuestionIndex"
-                                                  [totalQuestions]="mockQuestions.length"
-                                                  [progressPercentage]="getProgressPercentage()"
-                                                  [isLastQuestion]="isCurrentQuestionLast()"
-                                                  [savedResponse]="getSavedResponse()"
-                                                  (previousQuestion)="previousQuestion()"
-                                                  (nextQuestion)="nextQuestion()"
-                                                  (submitAssessment)="submitAssessment()"
-                                                  (responseChange)="updateResponse($event)">
-  </oppia-certificate-assessment-conversation-skin>
+<oppia-titled-background-banner [title]="bannerTitleI18nKey"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)